.github/workflows/integration_test.yaml

```diff
@@ -5,8 +5,10 @@ on:
     branches:
       - main
   push:
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
     branches:
-      - main
+      - "*"
```

Comment:
Do we really want to run all the tests on all the pushes to all the branches?

Or only in pull requests to any branch?

Reply:
Good point, updated it to any PR